src/bosh-director/spec/unit/deployment_plan/stages/package_compile_stage_spec.rb

```diff
@@ -906,16 +905,12 @@ def compile_package_with_url_encrypt_stub(args)
       before do
```

Comment:
It looks like that we should remove the `describe 'the global blobstore'` completely here. The tests doesn't make much sense or do I miss something here?

Reply:
you're right. removed it.